dask_planner/src/expression.rs

```diff
@@ -301,6 +300,13 @@ impl PyExpr {
                 unimplemented!("ScalarSubquery")
             }
             Expr::IsNotNull(expr) => Ok(vec![PyExpr::from(*expr.clone(), self.input_plan.clone())]),
+            Expr::ScalarUDF { fun: _, args } => {
+                let mut exprs: Vec<PyExpr> = Vec::new();
+                for arg in args {
+                    exprs.push(PyExpr::from(arg.clone(), self.input_plan.clone()));
+                }
+                Ok(exprs)
+            }
```

Comment:
nit: functional is more concise here

```suggestion
            Expr::ScalarUDF { args, .. } => Ok(args
                .iter()
                .map(|arg| PyExpr::from(arg.clone(), self.input_plan.clone()))
                .collect()),
```